otter/test/test_cloud_client.py

```diff
@@ -188,7 +201,11 @@ def test_binds_url(self):
         Binds a URL from service config if it has URL instead of binding
         URL from service catalog
         """
-        self.service_configs[ServiceType.CLOUD_SERVERS]['url'] = 'myurl'
+        self.service_configs = update_in(
+            self.service_configs,
+            (ServiceType.CLOUD_SERVERS, 'url'),
+            lambda _: 'myurl'
+        )
```

Comment:
Yeah... Since I made it so self.service_configs is a module-level constant (so I could share it between test suites), I had to avoid mutating it. First I tried deepcopying it, but that didn't work because it turns out if you deepcopy a Twisted constant you can no longer use it for identity comparison. :-(

I realized another thing I could do would be to just make a module-level function that returns the dict. I'll make that change.